assets/static/js/dataset-submission.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
I did this because VS studio showed everything is "red"

Reply:
I have made my editor to ignore that folder, we could solve it by adding a .eslintignore file with "assets/static/js"